rules/transition_support.bzl

```diff
@@ -123,15 +116,6 @@ def _apple_rule_transition_impl(settings, attr):
 
 _supports_visionos = hasattr(apple_common.platform_type, "visionos")
 
-_bazel_version = get_bazel_version(bazel_version)
-_bazel_major_version = int(_bazel_version.major)
-
-# `--apple_compiler` was removed in https://github.com/bazelbuild/bazel/commit/1acdfc422e724b4fe12c7bf5248086ab514ec4be
-_supports_clo_apple_compiler = _bazel_major_version < 7
-
-# `--apple_grte_top` was removed in https://github.com/bazelbuild/bazel/commit/fb4106bdbd23c365337ea99704921ada7b86c2df
-_supports_clo_apple_grte_top = _bazel_major_version < 7
-
```

Comment:
These are both related and removed in Bazel and now rules_ios for the same reason as: https://github.com/bazelbuild/rules_apple/pull/2040